bluemira/base/parameter_frame/_parameter.py

```diff
@@ -133,10 +137,14 @@ def value(self) -> ParameterValueType:
     def value(self, new_value: ParameterValueType):
         self.set_value(new_value, source="")
 
+    def value_as(self, unit: Union[str, pint.Unit]) -> ParameterValueType:
+        """Return the current value in a given unit"""
+        return raw_uc(self.value, self.unit, unit)
```

Comment:
We potentially need a try/catch to deal with cases where the value is not numeric here. We don't want to be exposing `pint` exceptions